### PR TITLE
Static properties and methods

### PR DESCRIPTION
**Pull request by mahdiHash**
Please review and comment/commit if anything is wrong.

### CODE REVIEW

File: 1-js/09-classes/03-static-properties-methods/article.md [hunk 1]
```diff
@@ -63,17 +63,17 @@ articles.sort(Article.compare);
 alert( articles[0].title ); // CSS
 ```
 
-Here `Article.compare` stands "above" articles, as a means to compare them. It's not a method of an article, but rather of the whole class.
+اینجا `Article.compare` در «بالای» مقاله‌ها (articles) قرار دارد، به عنوان واسطه‌ای برای مقایسه آن‌ها. این متدی از یک مقاله نیست، بلکه برای کل کلاس است.
```
Comment (mahdiHash) [inline 1-js/09-classes/03-static-properties-methods/article.md:66]: This line needs more attention.

Comment (mahdyar) [reply 1-js/09-classes/03-static-properties-methods/article.md:66]:
```suggestion
اینجا `Article.compare` در «بالای» مقاله‌ها (articles) قرار دارد، به عنوان روشی برای مقایسه آن‌ها. این متدی برای یک مقاله(article) نیست، بلکه برای کل کلاس است.
```

**Review by mahdyar — CHANGES_REQUESTED** [review]
Please review the suggestion(s).

**Comment by mahdiHash** [comment]
Thank you @mahdyar 
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHash!

**Pull request merged**